greenplumpython/expr.py

```diff
@@ -293,61 +412,11 @@ def _serialize(self) -> str:
         return f"{self.operator}({right_str})"
 
 
-class TypeCast(Expr):
-    """
-    Inherited from Expr. Representation of a Type Casting.
-    """
-
-    def __init__(
-        self,
-        obj: object,
-        type_name: str,
-        as_name: Optional[str] = None,
-        db: Optional[Database] = None,
-    ) -> None:
-        """
-
-        Args:
-            obj: object : which will be applied type casting
-            type_name : str : name of type which object will be cast
-        """
-        table = obj.table if isinstance(obj, Expr) else None
-        super().__init__(as_name, table, db)
-        self._obj = obj
-        self._type_name = type_name
-
-    def _serialize(self) -> str:
-        obj_str = self._obj._serialize() if isinstance(self._obj, Expr) else to_pg_const(self._obj)
-        return f"{obj_str}::{self._type_name}"
-
-
-class Type:
-    """
-    A Type object in Greenplum Database.
-    """
-
-    def __init__(self, name: str, db: Database) -> None:
-        self._name = name
-        self._db = db
-
-    def __call__(self, obj: object) -> TypeCast:
-        return TypeCast(obj, self._name, db=self._db)
-
-
-# FIXME: Rename gp.table(), gp.function(), etc. to get_table(), get_function(), etc.
-# FIXME: Make these functions methods of a Database,
-#  e.g. from gp.get_type("int", db) to db.get_type("int")
-def get_type(name: str, db: Database) -> Type:
-    """
-    Returns the type corresponding to the name in the database given.
-    """
-
-    return Type(name, db=db)
-
-
 class Column(Expr):
     """
-    Inherited from Expr. Representation of a python object Column.
+    Inherited from Expr.
+
+    Representation of a python object Column.
```

Comment:
```suggestion
    Representation of a Python object Column.
```

Reply:
Which means?

Reply:
Do you mean like `Representation of a Python object :class:`.Column`.`